build-logic/plugins/src/main/kotlin/AndroidCoordinates.kt

```diff
@@ -26,7 +27,15 @@ object AndroidSdk {
 object AndroidApp {
```

Comment:
this whole class need refactoring but i did not want to go deep into the rabbit hole, this way it will continue to work as before